src/lib/dataset/helpers.ts

```diff
@@ -76,7 +76,7 @@ function localUniqueStats(dataset: DataSet) {
  * item's rank is lesser than the other, any positive value otherwise.
  */
 function _statCompare(stat1: ColumnValueStat, stat2: ColumnValueStat) {
-  return stat2.count - stat1.count;
+  return JSON.stringify(stat1.value).localeCompare(JSON.stringify(stat2.value));
```

Comment:
Nice to use localeCompare there!
Some tips that can be useful to have a better sort:
- explicit the locale parameter to 'en', to avoid using the browser's locale which could lead to unreliable results
- try {sentityvity: 'base'}: it will collide upper and lower case letters, and diacritics (accents) a well, so it could be interesting (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator) 
- you can test the option {numeric: true} so numerical values would be correctly sorted (1 < 2 < 10 and not 1 < 10 < 2)  => test this against regular strings to see it doesn't break their sort (in that case better keep that off)

Reply:
Actually, sensitivity: "base", as lowercasing results, could create unpredictable orders.
Better to explicitly set which comes first using `caseFirst` upper or lower.